.travis.yml

```diff
@@ -56,6 +57,12 @@ jobs:
             if [[ "$DRIVER" == "pcov" ]]; then
               pecl install pcov;
             fi
+          - INFECTION_FLAGS="${INFECTION_FLAGS} $(get_infection_pr_flags)"
```

Comment:
difference with the previous approach: now we build `INFECTION_FLAGS` directly no more concatenation further down in the process